content/docs/notebooks/submit-kubernetes.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Submit Kubernetes Resources"
+description = "Submitting Kubernetes resources from a Jupyter notebook"
+weight = 40
++++
+
+Kubeflow assigns the `jupyter-notebook` service account to the Jupyter notebook
+Pods. This service account is bound to the `jupyter-notebook` role, which has namespace-scoped permissions to the following Kubernetes resources:
+
+* Pods
+* Deployments
+* Services
+* Jobs
+* TFJobs
+* PyTorchJobs
+
+You can therefore create the above Kubernetes resources directly from your
+Jupyter notebook in Kubeflow. The Kubernetes 
+[`kubectl`(https://kubernetes.io/docs/tasks/tools/install-kubectl/)] 
```

Comment:
Incorrect Markdown in link, please change to:

[`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)

Reply:
Fixed.